tests/clean/basic/test.sh

```diff
@@ -4,6 +4,7 @@
 rlJournalStart
     rlPhaseStartSetup
         rlRun "root=/var/tmp/tmt"
+        rlRun "test_root=/var/tmp/test_root"
```

Comment:
It might be better to use a temporary directory, e.g. like here: https://github.com/teemtee/tmt/blob/main/tests/run/error/test.sh#L8

Reply:
yeah,updated^^